test/code/ContextCheckerTest.php

```diff
@@ -65,7 +65,7 @@ public function messageProvider()
             // multiple sections possible with always two forming a pair
             foreach (array_chunk($parts, 2) as $chunk) {
                 $messages = array_filter(explode("\n", $this->canonicalize($chunk[1])));
-                $tests[] = array($fullName, $chunk[0], $messages);
+                $tests[] = array($fullName, ltrim($chunk[0]), $messages);
```

Comment:
This was needed because of a probable new line that was present before the `<?php ` opening tags of each tests. So it was needed to remove it to avoid this error when using namespace in tests : `Namespace declaration statement has to be the very first statement in the script on line 3`